### ISSUE TITLE
Install error on windows : path must be a string

### ISSUE THREAD

**Issue by basarat** [travs/atom-package-dependencies#5]
Here is an atom package where I am trying to install `linter` : https://github.com/TypeStrong/atom-typescript/blob/ae8171bbcfc268c7257f0cf24b29819efec3b692/lib/main/index.js#L3-L4 
```js
var apd = require('atom-package-dependencies');
apd.install();
```
my `package.json`  https://github.com/TypeStrong/atom-typescript/blob/ae8171bbcfc268c7257f0cf24b29819efec3b692/package.json#L26-L30: 
```js
  "package-dependencies": {
      "linter": "//this field will be ignored right now"
  }
```

Callstack: 
```js
TypeError: path must be a string
  at TypeError (native)
  at Object.fs.openSync (fs.js:455:18)
  at Object.module.(anonymous function) [as openSync] (c:\Users\basaratsyed\AppData\Local\atom\app-0.172.0\resources\atom\common\lib\asar.js:422:20)
  at Object.fs.readFileSync (fs.js:313:15)
  at Object.fs.readFileSync (c:\Users\basaratsyed\AppData\Local\atom\app-0.172.0\resources\atom\common\lib\asar.js:329:27)
  at c:\Users\basaratsyed\.atom\packages\atom-typescript\node_modules\atom-package-dependencies\index.js:78:30
  at c:\Users\basaratsyed\.atom\packages\atom-typescript\node_modules\atom-package-dependencies\index.js:71:5
  at c:\Users\basaratsyed\.atom\packages\atom-typescript\node_modules\atom-package-dependencies\node_modules\witwip\witwip.js:51:23
  at find (c:\Users\basaratsyed\.atom\packages\atom-typescript\node_modules\atom-package-dependencies\node_modules\witwip\witwip.js:95:12)
  at c:\Users\basaratsyed\.atom\packages\atom-typescript\node_modules\atom-package-dependencies\node_modules\witwip\witwip.js:106:16
  at c:\Users\basaratsyed\.atom\packages\atom-typescript\node_modules\atom-package-dependencies\node_modules\witwip\node_modules\read-package-json\read-json.js:80:40
  at c:\Users\basaratsyed\.atom\packages\atom-typescript\node_modules\atom-package-dependencies\node_modules\witwip\node_modules\read-package-json\read-json.js:128:48
  at c:\Users\basaratsyed\.atom\packages\atom-typescript\node_modules\atom-package-dependencies\node_modules\witwip\node_modules\read-package-json\node_modules\graceful-fs\graceful-fs.js:170:5
  at c:\Users\basaratsyed\.atom\packages\atom-typescript\node_modules\atom-package-dependencies\node_modules\witwip\node_modules\read-package-json\node_modules\graceful-fs\graceful-fs.js:179:5
  at fs.js:295:14
  at Object.oncomplete (fs.js:93:15)
```

--- 

Seems like a bug in `witwip` as shown below after analysis of the stacktrace
```
function getPackageJSONpath(callback){
  //returns path to 'package.json' in the Atom package that this is required by
  var findPkg = require('witwip');
  console.log('parent',module.parent); // correctly returns Module `c:\Users\basaratsyed\.atom\packages\atom-typescript\lib\main\index.js`
  findPkg(module.parent, function(err, pkgPath, pkgData) {
    var path = pkgPath; 
    console.log('path',path) // Probably the ERROR: path is undefined
    callback(path);
  });
}
```

**Comment by basarat** [comment]
Okay figured it out, its because `witwip` doesn't use `path.sep`

**Comment by basarat** [comment]
Since its published on `1.0.4` and we have `"witwip": "1.0.x",` so fixed!
